packages/testing/src/test/ContentBuilder.test.ts

```diff
@@ -242,6 +242,9 @@ describe("ContentBuilder", () => {
       presentationManagerMock.setup(async (manager) => manager.getContent(moq.It.isAny())).returns(getEmptyContent);
       presentationManagerMock.setup((x) => x.onIModelContentChanged).returns(() => new BeEvent());
       sinon.stub(Presentation, "presentation").get(() => presentationManagerMock.object);
+      sinon.stub(IModelApp, "quantityFormatter").get(() => ({
+        onActiveFormattingUnitSystemChanged: new BeUiEvent<FormattingUnitSystemChangedArgs>(),
```

Comment:
this suggests we're subscribing for the event somewhere in these tests - do you know where?

Reply:
This is where we [create default ContentDataProvider](https://github.com/iTwin/presentation/blob/4416ca4f1fa7d0f6127c7e651ffd23a7460b69af/packages/testing/src/presentation-testing/ContentBuilder.ts#L98) and this is the [test that uses it](https://github.com/iTwin/presentation/blob/4416ca4f1fa7d0f6127c7e651ffd23a7460b69af/packages/testing/src/test/ContentBuilder.test.ts#L252)

Reply:
looks like a useless test that tests the provider rather than the builder - maybe just remove it?..

Reply:
Refactored that test into two to make sure that test coverage is still at 100%.